### PR TITLE
repos: Update to RHEL 9.2 GA repos for OCP 4.14

### PR DESCRIPTION
**Pull request by HuijingHei**

### CODE REVIEW

**Comment by travier** [comment]
If the tests pass here then that should be fine.

File: core-services/release-controller/_repos/ocp-4.14-rhel92.repo [hunk 1]
```diff
@@ -36,7 +36,7 @@ failovermethod = priority
 
 [rhel-9.2-nfv]
 name = rhel-9.2-nfv
-baseurl = https://cdn.redhat.com/content/e4s/rhel9/9.2/x86_64/nfv/os/
```
Comment (travier) [inline core-services/release-controller/_repos/ocp-4.14-rhel92.repo:39]: The naming has changed for RHEL 9 and we need to keep `e4s` here. `tus` was the old name for RHEL 8.

Comment (HuijingHei) [reply core-services/release-controller/_repos/ocp-4.14-rhel92.repo]: Updated, thanks!

**Review by travier — APPROVED** [review]
/lgtm
/approve

**Review by sosiouxme — APPROVED** [review]
/lgtm

**Pull request merged**